pkg/libstack/compose/compose_test.go

```diff
@@ -16,9 +15,9 @@ import (
 )
 
 func checkPrerequisites(t *testing.T) {
-	if _, err := os.Stat("docker-compose"); errors.Is(err, os.ErrNotExist) {
-		t.Fatal("docker-compose binary not found, please run download.sh and re-run this suite")
-	}
```

Comment:
Why was this commented out?  is it needed?

Reply:
actually, with this one it I couldn't run the tests, because I don't have the docker-compose binary. compose is built-in in docker. as we can see, the tests work without this check